content/ADR-138-social-service-authentication.md

```diff
@@ -0,0 +1,101 @@
+---
+adr: 138
```

Comment:
Please use ADR id same as PR (in this case 143) to avoid conflicts with other ADRs (there's already another one ongoing with same id).